hypershiftoperator/deploy/templates/operator.servicemonitor.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: azmonitoring.coreos.com/v1
```

Comment:
why is this entirely deleted?

Reply:
The hypershift operator lays down its own service monitor, which didn't work before since it wasn't using azmonitor.coreos.com.